### ISSUE TITLE
Indentation in decorators not working properly

### ISSUE THREAD

**Issue by JamesHenry** [typescript-eslint/typescript-eslint#9]
This issue was initially reported here: https://github.com/eslint/typescript-eslint-parser/issues/438

---

**What version of TypeScript are you using?**
2.6.2

**What version of `typescript-eslint-parser` are you using?**
12.0.0

**What code were you trying to parse?**

```ts
import { Component, Vue } from 'vue-property-decorator'
import TForm from '@components/t-form.vue'

@Component({
  components: {
    't-form': TForm
  }
})
export default class extends Vue {
```

**What did you expect to happen?**
No linting errors

**What happened?**
Got the following errors

```ts
import { Component, Vue } from 'vue-property-decorator'
import TForm from '@components/t-form.vue'

@Component({
  components: {
    't-form': TForm    <--- *** Expected indentation of 2 spaces but found 4  indent ***
  }
})    <--- *** Expected indentation of 2 spaces but found 0  indent ***
export default class extends Vue {
```

**Comment by meyer** [comment]
@JamesHenry Not related to this particular issue, but I believe you can preserve the original issue comments by using the (beta) issue migration functionality that was added to Github a few months ago. Looks something like this:

![screenshot 2019-01-15 15 47 17](https://user-images.githubusercontent.com/13781/51217442-fabbe380-18dc-11e9-99a7-dcf651549935.png)

It might require you to (temporarily) un-archive the source repo. Also I think you might need certain permissions on the destination repo? Not 100% sure. Here’s more to read if you’re curious: https://help.github.com/articles/transferring-an-issue-to-another-repository/

Sorry for the off-topic comment.

**Comment by bradzacher** [comment]
@meyer - that feature requires the repos to be within the same namespace (i.e. we would have had to move `typescript-eslint-parser` into `typescript/eslint`, move the issues, then move the repo back).

**Comment by armano2** [comment]
You can solve this issue by using `eslint-plugin-typescript/indent` rule instead of `eslint/indent`

[Test case](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin-typescript/tests/lib/rules/indent.js#L673)

**Comment by fatfatson** [comment]
problem still exist!
all fields with decorator won't have correct indent checking.

**Comment by bradzacher** [comment]
@fatfatson - could you please open a new issue and fill out the template?
Just throwing a comment on a super old issue saying that the problem still exists doesn't help us at all.
Having a quick look it looks like it should be working fine, considering there's a (passing) test case in master

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/tests/rules/indent/indent.test.ts#L676-L704

To reiterate: Please open a new issue.
